src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -774,14 +793,14 @@ def _load_file_string_or_uri(file_or_string_or_uri, name, required=True):
     return shell_safe_json_parse(file_or_string_or_uri)
 
 
-def create_policy_definition(name, rules=None, params=None, display_name=None, description=None):
+def create_policy_definition(name, rules=None, params=None, display_name=None, description=None, mode=None):
```

Comment:
@derekbekoe  `mode` is only available after 2016-12-01. I'm not if there is something to do here or it's ok.

Reply:
@lmazuel - mode is available on the old api version (2016-12-01) if that's what you mean

Reply:
@vivsriaus, yes I mean from 2016-12-01 included, thus not 2015-10-01-preview (Stack version)

Reply:
@Imazuel so there shouldn't be any changes for stack here, correct? @derekbekoe

Reply:
@vivsriaus If this is exposed as a --mode parameter for the CLI, this is has to be declared only compatible with >=2016-12-01. My comment is really a triple-check to be sure everything is fine, I don't know the CLI enough to be sure myself it will not be exposed when the Profile is set to Stack.

Reply:
@vivsriaus, please apply api-version condition in the params.py file around the `mode`

Reply:
We actually don't check for api version when mode is passed, so it is available for ALL api-versions

Reply:
@vivsriaus See @yugangw-msft comment, in old ApiVersion the class `PolicyDefinition` does not have `mode` and the `__init__` call will fail without the ApiVersion check.